docs/certification/conventions.md

```diff
@@ -0,0 +1,12 @@
+## Conventions
+
+The 0xcert Framework provides some basic schema conventions which you can use in your projects.
+
+| Issue | Id | Description
+|-|-|-
+| [88](https://0xcert.org/conventions/xcert-schema.html) | xxxx | JSON Schema definition describing asset schema capabilities.
```

Comment:
Link broken

Reply:
https://0xcert.org/conventions/xcert-schema.html

Reply:
What is XXXX

Reply:
This PR is tagged with `work in progress`